pages/donor-story.html

```diff
@@ -14,7 +14,7 @@
 image: img/katherine-harris-tom.jpg
 image-alt: "A white woman and a white man stand in front of the Washington Monument in Washington, D.C."
 image-caption: "Katherine and her husband, Tom Keyser in Washington, D.C. after attending the 2019 Ambassador Council gathering."
-recommendation-name: "If you like <em>This American Life</em>, you may also enjoy ..."
+recommendation-name: "If you like <em><a href='https://thisamericanlife.org' target='_blank'><span>This American Life</span></a></em>, you may also enjoy ..."
```

Comment:
Recommend updating to:

`<a href='https://thisamericanlife.org' target='_blank'><em>This American Life</em></a>`

That way the italics are contained within the anchor link.